lib/mod.ts

```diff
@@ -56,7 +48,7 @@ export async function hash(
   password: string,
   options: Partial<HashOptions> = {},
 ) {
-  return plugin.hash(password, options);
+  return await internal.hash(password, options);
```

Comment:
I believe we should be able to remove the `await` here and the `async` in the function signature as well.